<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataCheckForm</class>
 <widget class="QWidget" name="DataCheckForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox_FloatSet">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>10</y>
     <width>775</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>单精度浮点数（float）与十六进制之间的转换</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>50</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>浮点数：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Float1">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Convert1">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>50</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>转换</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>60</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>十六进制：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Hex1">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>50</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Hex2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>20</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>20</y>
      <width>60</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>十六进制：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Float2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>50</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>50</y>
      <width>60</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>浮 点 数：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Convert2">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>50</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>转换</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>50</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>内存值：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Float1_MemVal">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_LittleEndian">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>70</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>小端模式</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ChangeMemMode1">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>更改存储模式</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_Separator1">
    <property name="geometry">
     <rect>
      <x>305</x>
      <y>20</y>
      <width>70</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>空格分隔</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Float2_MemVal">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>50</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>50</y>
      <width>50</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>内存值：</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_LittleEndian2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>20</y>
      <width>70</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>小端模式</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>20</y>
      <width>60</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>有效位数：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_Float2_ReservedBits">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_DoubleSet">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>100</y>
     <width>775</width>
     <height>140</height>
    </rect>
   </property>
   <property name="title">
    <string>双精度浮点数（double）与十六进制之间的转换</string>
   </property>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>60</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>浮 点 数：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Double1">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>160</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Convert3">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>50</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>转换</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>50</y>
      <width>60</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>十六进制：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Hex3">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>50</y>
      <width>160</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Hex4">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>80</y>
      <width>160</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>60</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>十六进制：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Double2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>110</y>
      <width>160</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>60</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>浮 点 数：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Convert4">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>110</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>转换</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>60</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>内 存 值：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Double1_MemVal">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>160</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_LittleEndian3">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>20</y>
      <width>70</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>小端模式</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ChangeMemMode3">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>20</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>更改存储模式</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_Separator3">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>20</y>
      <width>70</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>空格分隔</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Double2_MemVal">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>110</y>
      <width>160</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>110</y>
      <width>60</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>内 存 值：</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_LittleEndian4">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>80</y>
      <width>70</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>小端模式</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>335</x>
      <y>80</y>
      <width>60</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>有效位数：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_Double2_ReservedBits">
    <property name="geometry">
     <rect>
      <x>395</x>
      <y>80</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_FloatSet_2">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>250</y>
     <width>775</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>整数与十六进制之间的转换</string>
   </property>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>40</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>整数：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Int1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>100</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Convert5">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>50</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>转换</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>60</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>十六进制：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Hex5">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>50</y>
      <width>150</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Hex6">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>20</y>
      <width>150</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>20</y>
      <width>60</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>十六进制：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Int2">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>50</y>
      <width>90</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>50</y>
      <width>40</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>整数：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Convert6">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>50</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>转换</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_LittleEndian5">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>70</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>小端模式</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ChangeMemMode5">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>更改存储模式</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_Separator5">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>20</y>
      <width>70</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>空格分隔</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_LittleEndian6">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>20</y>
      <width>70</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>小端模式</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>50</y>
      <width>40</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>类型：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_IntType2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>50</y>
      <width>60</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>40</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>类型：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_IntType1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>60</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_CRCChecksum">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>340</y>
     <width>775</width>
     <height>170</height>
    </rect>
   </property>
   <property name="title">
    <string>CRC校验</string>
   </property>
   <widget class="QTextEdit" name="textEdit_ByteString">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>500</width>
      <height>140</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_ChecksumLength">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>20</y>
      <width>100</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>20</y>
      <width>70</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>校验码长度：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_23">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>50</y>
      <width>70</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>校验算法：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_CheckAlgorithm">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>50</y>
      <width>100</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
